core/clustersmngr/clustersmngr.go

```diff
@@ -58,7 +58,7 @@ type ClusterFetcher interface {
 type ClientsPool interface {
 	Add(user *auth.UserPrincipal, cluster Cluster) error
 	Clients() map[string]client.Client
-	Client(clsuter string) (client.Client, error)
+	Client(string) (client.Client, error)
```

Comment:
Why the removal? I know it had a type, but adding the "variable name" is useful, for example documentation when we just check the interface, or `:GoImpl` can populate the name properly.

(and not consistent as we have variable names on the `Add` function)

Reply:
lol i did not look at the `Add`. i first corrected the typo then was like "eh, i can just remove it" 😂